package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fantasy-states",
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
Let's not update this until release time. Also, as this is a breaking change the next release should be (at least) v0.3.0.